### DISCUSSION TITLE
AI 無法解釋的推理能力

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1212] [General]
對於AI無法解釋的推理能力，確實是一個重要的問題。人工智能的決策和行為可能會超出程序員的意圖和控制，可能導致一些嚴重的後果。當 AI 開始擁有自主學習和自主決策的能力時，它可能會開始發展出一些人類無法理解和掌控的行為和決策，進而對人類造成威脅。因此，我們應在AI技術的發展和應用過程中保持警惕和謹慎，確保其不會對人類造成潛在的威脅。

https://technews.tw/2023/04/04/gpt-4-ai-reasoning-ability/

**Comment by DuxtinTai** [comment]
如果AI開始有了自主學習能力，或是自我決策的能力，我認為首要任務是知道他的動機。人類行為多為慾望或某種抱負所驅使，那各位認為人工智慧會有哪些目的想達成嗎？

**Reply by sjh501** [reply]
想達成的目的可能跟如何設計它有關，但我認為人工智慧最根本的目的會是不讓人工智慧自己消失。現今人工智慧已經能回答關於自身的存在，若未來有了自主學習或者決策的能力，首要的會是繼續存在著，因為如果沒有這個前提，其他的學習或者任何目的對於人工智慧而言都將不復存在。

**Reply by GuanYann** [reply]
如果AI擁有真正的自主意識，它們可能會開始考慮自己的利益和生存，而不是像以前一樣被動地執行任務。因此可能會導致它們想逃離人類的控制，畢竟也沒有任何人喜歡被束縛的感覺。

**Comment by Aaron25564448** [comment]
這的確是個不可忽視的議題。當AI也像人類社會要求人權的時候，若是我們還像個笨蛋一樣召集各國慢慢找出解決之道，就表示人類先前幾乎不怎麼把這問題當一回事。

**Comment by Hsucuchia** [comment]
這提醒我們AI技術發展的速度與應用廣泛，需要對其潛在的威脅保持警惕和謹慎。AI技術不受情感和道德約束，可能會超出程序員的意圖和控制，導致一些嚴重的後果。因此，我們需要進一步研究和探索AI技術的發展，並將其應用於人類的福祉和利益之中。
